.eslintrc

```diff
@@ -151,37 +181,62 @@
       }
     },
     {
-      "files": ["packages/meditrak-app/**"],
-      "plugins": ["react-native"],
+      "files": [
+        "packages/meditrak-app/**"
+      ],
+      "plugins": [
+        "react-native"
+      ],
       // "env": {
       //   "react-native/react-native": true
       // },
       "rules": {
         "react/destructuring-assignment": "off",
-        "react/prop-types": ["error", { "ignore": ["navigation"] }]
+        "react/prop-types": [
+          "error",
+          {
+            "ignore": [
+              "navigation"
+            ]
+          }
+        ]
       }
     },
     {
-      "files": ["packages/ui-components/**"],
+      "files": [
+        "packages/ui-components/**"
+      ],
       "rules": {
-        "import/no-extraneous-dependencies": "off"
+        "import/no-extraneous-dependencies": "off",
+        "react/prop-types": "off",
+        "import/no-unresolved": 0, // Turn off linting error for imports
+        "import/extensions": 0, // Turn off linting error for imports
+        "react/require-default-props": "off"
```

Comment:
I assume these are the only changes and the rest is just auto formatting, right?

Reply:
I believe so, yes!